resources/views/cli.blade.php

```diff
@@ -5,11 +5,14 @@
     <x-container>
 
 
-        <x-title>CLI</x-title>
-
-        <x-subtitle>
+        <x-header>CLI</x-header>
+        <ul>
+            <x-context>TBD</x-context>
+        </ul>
+        <x-Title>
```

Comment:
@the-zalox you seem to have x-title instead of x-Title in other places. Is this intentional?

Reply:
@Frank-Landry  I have sorted this for my next PR. doesn't change anything just making it more standard.

Reply:
@the-zalox 👍